documentation/config_file_auth_migration.markdown

```diff
@@ -8,11 +8,11 @@ canonical: "/puppetserver/latest/config_file_auth_migration.html"
 [deprecated]: ./deprecated_features.markdown
 [Backward Compatibility with Puppet 3 Agents]: ./compatibility_with_puppet_agent.markdown
 
-Puppet Server 2.2.0 introduced a significant change in how it manages authentication to API endpoints. The older [Puppet `auth.conf`][] file and whitelist-based authorization method are [deprecated][]. Puppet Server's new `auth.conf` file, illustrated below in examples, also uses a different format for authorization rules.
+Puppet Server 2.2.0 introduced a significant change in how it manages authentication to API endpoints. The older [Puppet `auth.conf`][] file and whitelist-based authorization method are [deprecated][]. Puppet Server's new `auth.conf` file, illustrated below in examples, also uses a different format for authorization rules. Puppet Server removed the ability to switch back to the legacy mode, so you must upgrade all of your old rules when moving to that version.
```

Comment:
How about something more like:
"Puppet Server 2.2.0 introduced a significant change in how it manages authentication to API endpoints. The older [Puppet `auth.conf`][] file and whitelist-based authorization method were [deprecated][] in the same release and are now removed in Puppet Server 7. Puppet Server's current `auth.conf` file format (which is different than the old auth.conf) is illustrated below in examples."